### ISSUE TITLE
Make proposal variance for edge moves within topology moves depend on current branch lengths

### ISSUE THREAD

**Issue by 1j** [statalign/statalign#39]

**Comment by bredelings** [comment]
This sounds like it wouldn't satisfy detailed balance very well.  At least you'd need to consider that the initial state and the proposed state could used different variances.  If you use a lognormal proposal in addition to a normal proposal, then it should automatically propose larger changes to larger branches.

**Comment by 1j** [comment]
Yes I think that was the intention here -- probably ambiguous wording in the issue title. I meant that the proposal variance encoded by `edgeProposalWidthControlVariable` in `statalign.base.mcmc.TopologyMove` should have a different effect on each branch depending on its current length. One way to incorporate that to some extent would be using `MultiplicativeProposal` (i.e. lognormal) in addition to `UniformProposal`, which was originally in the code (currently commented out), but was removed because the `UniformProposal` was mixing better for individual `EdgeMove`s . I think my original plan had been to have one proposal variance to generate a single auxiliary variable, and to split this proportionally among the different edges, which is not dissimilar to what happens with `LOCALTopologyMove`, but I didn't get a chance to work through the details so alas this one was left hanging for a while (wow, almost 7 years...!)
